script.js

```diff
@@ -1,4 +1,190 @@
+  return `Hello Player 1 🐶.<br><br>You rolled ${playerOneDiceOne} for Dice 🎲 One and ${playerOneDiceTwo} for Dice 🎲 Two.<br><br>Now please choose the order of your Dice 🎲 by entering 1 or 2.`;
+};
+
+// player 1 choose dice position
+var playerOneChoice = function (input) {
+  if (input == 1 || input == 2) {
+    playerOneNumber = combineDiceRolls(input);
+    playerOneScore.push(playerOneNumber);
+    gameMode = "player 2";
+    return `Player 1 🐶.<br><br>Your chosen number is ${playerOneNumber}.<br><br>It is now Player 2's 🐹 turn to give it a go.<br><br>Click submit to roll the dice 🎲.`;
+  } else {
+    return verifyInput(input);
+  }
+};
+
+// player 2 dice roll
+var playerTwoDiceRoll = function () {
+  playerTwoDiceOne = rollDice();
+  playerTwoDiceTwo = rollDice();
+  gameMode = "player 2 position";
+  return `Hello Player 2 🐹.<br><br>You rolled ${playerTwoDiceOne} for Dice 🎲 One and ${playerTwoDiceTwo} for Dice 🎲 Two.<br><br>Now please choose the order of your Dice 🎲 by entering 1 or 2.`;
+};
+
+// player 2 choose dice position
+var playerTwoChoice = function (input) {
+  if (input == 1 || input == 2) {
+    playerTwoNumber = combineDiceRolls(input);
+    playerTwoScore.push(playerTwoNumber);
+    gameMode = "score time";
+    return `Player 2 🐹.<br><br>Your chosen number is ${playerTwoNumber}.<br><br>Let's see who's the winner?! 🍾`;
+  } else {
+    return verifyInput(input);
+  }
+};
+
+// verify if input is a valid number
+var verifyInput = function (input) {
+  if (input != 1 || input != 2) {
+    return "🙅 Please enter 1 or 2 to position your dice 🎲 roll.";
+  }
+};
+
+// combine the dice rolls
+var combineDiceRolls = function (input) {
+  if (input == 1 && gameMode == "player 1 position") {
+    return playerOneDiceOne + playerOneDiceTwo;
+  } else if (input == 1 && gameMode == "player 2 position") {
+    return playerTwoDiceOne + playerTwoDiceTwo;
+  } else if (input == 2 && gameMode == "player 1 position") {
+    return playerOneDiceTwo + playerOneDiceOne;
+  } else if (input == 2 && gameMode == "player 2 position") {
+    return playerTwoDiceTwo + playerTwoDiceOne;
+  }
+};
+
+// player scoring
+var finalScore = function () {
+  if (playerOneScore > playerTwoScore) {
+    return `Player 1 🐶 wins 🏆 with ${
+      playerOneScore[playerOneScore.length - 1]
+    } and Player 2 🐹 loses 😭 with ${
+      playerTwoScore[playerTwoScore.length - 1]
+    }`;
+  } else if (playerOneScore < playerTwoScore) {
+    return `Player 1 🐶 loses 😭 with ${
+      playerOneScore[playerOneScore.length - 1]
+    } and Player 2 🐹 wins 🏆 with ${
+      playerTwoScore[playerTwoScore.length - 1]
+    }`;
+  } else if (playerOneScore == playerTwoScore) {
+    return `It is a draw as Player 1 🐶 number is ${
+      playerOneScore[playerOneScore.length - 1]
+    } and Player 2 🐹 number is also ${
+      playerTwoScore[playerTwoScore.length - 1]
+    }`;
+  }
+};
+
+// continue playing
+var playAgain = function () {
+  gameMode = "player 1";
+};
```

Comment:
I like that you have a reset function! It makes it clear what variables control the game